src/styles/block/swiper.scss

```diff
@@ -0,0 +1,19 @@
+.swiper {
+
+  @include onDesktop {
+    display: none;
+  }
+
+  &-pagination {
+
+    padding-top: 20px;
+
+    @include onTablet {
+      margin-top: 32px;
+    }
+
+    &-bullet-active {
+      color: green;
+    }
+  }
```

Comment:
Why. do you use such a nesting here `&-`?

You could nest modifiers, or elements, but there you have non of them.

Watch more about naming convention in BEM [in this video](https://mate.academy/learn/css-advanced/bem#/video/920)
Watch closely when you use '-' in naming.

Reply:
Thanks! It was a piece of code (both HTML and CSS) from the swiper library! So I did not know whether I could change the default naming to modify styles. But I will handle it!)

Reply:
If it's from a swiper library or smt like that, you could always write in comment about it and we could discuss is it important to change.

Reply:
Thanks!